torch/csrc/api/include/torch/nn/pimpl.h

```diff
@@ -57,12 +57,8 @@ class ModuleHolder : torch::detail::ModuleHolderIndicator {
       typename Head,
       typename... Tail,
       typename = typename std::enable_if<
-      !(
-          torch::detail::is_module_holder_of<Head, ContainedType>::value
-          && (sizeof...(Tail) == 0)
-       )
-      >::type
-          >
+          !(torch::detail::is_module_holder_of<Head, ContainedType>::value &&
+            (sizeof...(Tail) == 0))>::type>
```

Comment:
uhhh... did this actually change?

Reply:
No it's  just reformatted. I don't know why it wasn't formatted